.github/workflows/jerigon-native.yml

```diff
@@ -58,41 +57,37 @@ jobs:
 
       #It is much easier to use cast tool in scripts so install foundry
       - name: Install Foundry
-        uses: foundry-rs/foundry-toolchain@v1          
+        uses: foundry-rs/foundry-toolchain@v1
 
       - name: Run cancun test network
         run: |
           docker pull ghcr.io/0xpolygonzero/erigon:feat-zero
-          kurtosis run --enclave cancun-testnet github.com/ethpandaops/ethereum-package@4.0.0 --args-file jerigon-test-network/network_params.yml  
+          kurtosis run --enclave cancun-testnet github.com/ethpandaops/ethereum-package@4.0.0 --args-file jerigon-test-network/network_params.yml
 
       - name: Generate blocks with transactions
         run: |
-            ETH_RPC_URL="$(kurtosis port print cancun-testnet el-2-erigon-lighthouse ws-rpc)"
-            cast rpc eth_blockNumber --rpc-url $ETH_RPC_URL
-            cd jerigon-test-network && set -a && source .env && set +a
-            bash ./tests/generate_transactions.sh
- 
+          ETH_RPC_URL="$(kurtosis port print cancun-testnet el-2-erigon-lighthouse ws-rpc)"
```

Comment:
What formater have you used? I don't mind, just curious.

Reply:
I am using some VScode formatter. If this is not standard practice, i can check it and fix it. Let me know